### PR TITLE
Feature #180537199 – Import proteomics differential experiments

### PR DESCRIPTION
**Pull request by alfonsomunozpomer**
Add a case to write the experiment design file for proteomics differential experiments. I added a test and an entry in the experiment fixture, too.

### CODE REVIEW

File: src/main/java/uk/ac/ebi/atlas/experimentimport/experimentdesign/ExperimentDesignFileWriter.java [hunk 1]
```diff
@@ -76,6 +76,7 @@ public void write(ExperimentDesign experimentDesign) throws IOException {
             case RNASEQ_MRNA_BASELINE:
             case RNASEQ_MRNA_DIFFERENTIAL:
             case PROTEOMICS_BASELINE:
+            case PROTEOMICS_DIFFERENTIAL:
```
Comment (upendrakumbham) [inline src/main/java/uk/ac/ebi/atlas/experimentimport/experimentdesign/ExperimentDesignFileWriter.java:79]: do we need to add the 'DIA' experiment case here?

Comment (alfonsomunozpomer) [reply src/main/java/uk/ac/ebi/atlas/experimentimport/experimentdesign/ExperimentDesignFileWriter.java:79]: Well spotted.

Comment (alfonsomunozpomer) [reply src/main/java/uk/ac/ebi/atlas/experimentimport/experimentdesign/ExperimentDesignFileWriter.java:79]: Fixed.

**Review by upendrakumbham — APPROVED** [review]
Lots of code cleanup and all changes are good. Ran a test class `ExperimentDesignFileWriterTest.java` in my local. all tests passing after DIA case addition.

**Pull request merged**